### ISSUE TITLE
tikv-ctl should be included in the tikv docker image

### ISSUE THREAD

**Issue by DanielZhangQD** [tikv/tikv#7756]
### Is your feature request related to a problem? Please describe:

There is no dedicated download link for tikv-ctl and no binary in the docker image, it's not easy for Kubernetes users to get the tikv-ctl binary.
### Describe the feature you'd like:

tikv-ctl should be included in the tikv docker image.

**Comment by overvenus** [comment]
TiKV docker image does include tikv-ctl.

https://github.com/tikv/tikv/blob/2d6acde569ec1246b5baa6dcda70fbafab1ba886/scripts/gen-dockerfile.sh#L110-L111

**Comment by DanielZhangQD** [comment]
I checked both v3.1.0-rc and v4.0.0-rc images, the tikv-ctl is not included in the image, Please help check further.
```
# k get pod test31-tikv-0 -n test31 -o yaml | grep image:
    image: pingcap/tikv:v3.1.0-rc
    image: pingcap/tikv:v3.1.0-rc

# k exec -it test31-tikv-0 -n test31 sh
/ # ls
bin          etc          lib          media        opt          root         sbin         sys          tmp          var
dev          home         lib64        mnt          proc         run          srv          tikv-server  usr
/ # find / -name tikv-ctl
/ # 

# k get pod test40-tikv-0 -n test40 -o yaml | grep image:    
    image: pingcap/tikv:v4.0.0-rc
    image: pingcap/tikv:v4.0.0-rc

# k exec -it test40-tikv-0 -n test40 sh    
/ # ls
bin          etc          lib          media        opt          root         sbin         sys          tmp          var
dev          home         lib64        mnt          proc         run          srv          tikv-server  usr
/ # find / -name tikv-ctl
/ # 
```

**Comment by MyonKeminta** [comment]
tikv-ctl is missing in the already released docker images. It will be included in the future (3.1.2, 3.0.14, 4.0.0-rc.2 and later versions). Also we will publish images for tikv-ctl 3.0.12 and tikv-ctl 3.1.1 (images that contains only tikv-ctl).

**Comment by DanielZhangQD** [comment]
> tikv-ctl is missing in the already released docker images. It will be included in the future (3.1.2, 3.0.14, 4.0.0-rc.2 and later versions). Also we will publish images for tikv-ctl 3.0.12 and tikv-ctl 3.1.1 (images that contains only tikv-ctl).

Thanks for the update!
